app/views/miq_ae_customization/_dialog_field_form_non_dynamic_options.html.haml

```diff
@@ -110,6 +110,16 @@
       :javascript
         miqSelectPickerEvent('field_required', "#{url}")
 
+  - if %w(DialogFieldDropDownList).include?(@edit[:field_typ])
+    .form-group
+      %label.col-md-2.control-label
+        = _('Multiselect')
+      .col-md-8
+        = check_box_tag('field_multi_value', true,
+                        @edit[:field_multi_value],
+                        :disabled                   => disable_check_box?,
```

Comment:
`disable_check_box?` is a method related to `DialogFieldTagControl`, so I don't think it applies here, plus probably should never be disabled since then they wouldn't be able to set the field.